plugins/jobs/src/js/jobsToggleSchedule.js

```diff
@@ -0,0 +1,22 @@
+import { updateSchedule } from "#SRC/js/events/MetronomeClient";
```

Comment:
🎨 no `.ts` 😭 (this file)

Reply:
Also: shouldnt it be `jobToggleSchedule`? 🤔 For clarification I'd love to see these files name `JobsMenuToggleSchedule` or similar 🤔

Reply:
please look at the jobToggleSchedule answer.

Reply:
please look at the jobsDelete answer.